action/lib/common.js

```diff
@@ -130,6 +130,30 @@ function deleteTrigger(endpoint, params, actionName) {
         });
 }
 
+function getTrigger(endpoint, params, actionName) {
+    var options = {
+        method: 'GET',
+        url: getWebActionURL(endpoint, actionName),
+        rejectUnauthorized: false,
```

Comment:
What's `rejectUnauthorized` do?

Reply:
not entirely sure. i think it might have something to do with accepting self-signed certs https://github.com/request/request-promise/issues/225

Reply:
not entirely sure though